boards with i2c child nodes fail to build on windows

When building on Windows, `extract_dts_includes.py` fails with an exception on boards that have i2c child nodes in dts.

Boards affected:
- disco_l475_iot1
- frdm_k64f
- frdm_kw41z
- hexiwear_k64

```
C:\Users\nxa17364\zephyr\samples\hello_world\build>cmake -DBOARD=frdm_k64f -GNinja ..
CMake Deprecation Warning at C:/Users/nxa17364/zephyr/cmake/app/boilerplate.cmake:38 (cmake_policy):
  The OLD behavior for policy CMP0000 will be removed from a future version
  of CMake.

  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.
Call Stack (most recent call first):
  CMakeLists.txt:2 (include)


-- Found PythonInterp: C:/Python36/python.exe (found suitable version "3.6.4", minimum required is "3.4")
-- Selected BOARD frdm_k64f
Zephyr version: 1.11.99
warning: HAS_DTS_I2C_DEVICE (defined at C:/Users/nxa17364/zephyr/dts/Kconfig:22) has unsatisfied direct dependencies (HAS_DTS_I2C), but is currently being selected by the following symbols:
BOARD_FRDM_K64F (defined at C:/Users/nxa17364/zephyr/boards/arm/frdm_k64f/Kconfig.board:2), with value y, direct dependencies SOC_SERIES_KINETIS_K6X && <choice> (value: y), and select condition SOC_SERIES_KINETIS_K6X && <choice> (value: y)
Parsing Kconfig tree in C:/Users/nxa17364/zephyr//Kconfig
Using C:/Users/nxa17364/zephyr/boards/arm/frdm_k64f/frdm_k64f_defconfig as base
Merging C:/Users/nxa17364/zephyr/samples/hello_world/prj.conf
-- Generating zephyr/include/generated/generated_dts_board.h
Traceback (most recent call last):
  File "C:/Users/nxa17364/zephyr//scripts/dts/extract_dts_includes.py", line 918, in <module>
    main()
  File "C:/Users/nxa17364/zephyr//scripts/dts/extract_dts_includes.py", line 833, in main
    yaml_list[c] = yaml.load(yf, Loader)
  File "C:\Users\nxa17364\AppData\Roaming\Python\Python36\site-packages\yaml\__init__.py", line 72, in load
    return loader.get_single_data()
  File "C:\Users\nxa17364\AppData\Roaming\Python\Python36\site-packages\yaml\constructor.py", line 37, in get_single_data
    return self.construct_document(node)
  File "C:\Users\nxa17364\AppData\Roaming\Python\Python36\site-packages\yaml\constructor.py", line 46, in construct_document
    for dummy in generator:
  File "C:\Users\nxa17364\AppData\Roaming\Python\Python36\site-packages\yaml\constructor.py", line 398, in construct_yaml_map
    value = self.construct_mapping(node)
  File "C:\Users\nxa17364\AppData\Roaming\Python\Python36\site-packages\yaml\constructor.py", line 204, in construct_mapping
    return super().construct_mapping(node, deep=deep)
  File "C:\Users\nxa17364\AppData\Roaming\Python\Python36\site-packages\yaml\constructor.py", line 129, in construct_mapping
    value = self.construct_object(value_node, deep=deep)
  File "C:\Users\nxa17364\AppData\Roaming\Python\Python36\site-packages\yaml\constructor.py", line 86, in construct_object
    data = constructor(self, node)
  File "C:/Users/nxa17364/zephyr//scripts/dts/extract_dts_includes.py", line 117, in include
    return self.extractFile(self.construct_scalar(node))
  File "C:/Users/nxa17364/zephyr//scripts/dts/extract_dts_includes.py", line 145, in extractFile
    with open(filepath, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: ''
CMake Error at C:/Users/nxa17364/zephyr/cmake/dts.cmake:111 (message):
  command failed with return code: 1
Call Stack (most recent call first):
  C:/Users/nxa17364/zephyr/cmake/app/boilerplate.cmake:260 (include)
  CMakeLists.txt:2 (include)


-- Configuring incomplete, errors occurred!
```

--- a/scripts/dts/extract_dts_includes.py
+++ b/scripts/dts/extract_dts_includes.py
@@ -137,8 +137,7 @@ class Loader(yaml.Loader):
         if not os.path.isfile(filepath):
             # we need to look in bindings/* directories
             # take path and back up 1 directory and parse in '/bindings/*'
-            filepath = os.path.dirname(self._root).split('/')
-            filepath = '/'.join(filepath[:-1])
+            filepath = os.path.dirname(os.path.dirname(self._root))
             for root, dirnames, file in os.walk(filepath):
                 if fnmatch.filter(file, filename):
                     filepath = os.path.join(root, filename)

